server/routes/productRoutes.js

```diff
@@ -1,52 +1,49 @@
-const router = require('./index');
+const router = require('express').Router();
 const Product = require('../db/models/Product');
 
-router.get('/products', async (req, res, next) => {
-    try {
-        res.send(await Product.findAll());
-    } catch (err){
-        console.error(err);
-    }
-});
-
-router.post('/products', async (req, res, next) => {
-    try {
-        res.send(await Product.create(req.body));
-    } catch (err){
-        console.error(err);
-    }
-});
-
-router.delete('/products/:productId', async (req, res, next) => {
-    try {
-        res.send(await Product.destroy({
-        where: {
-            id: req.params.id
+router.route('/')
+    .get(async (req, res, next) => {
+        try {
+            res.send(await Product.findAll());
+        } catch (err){
+            console.error(err);
+        }
+    })
+    .post(async (req, res, next) => {
+        try {
+            res.send(await Product.create(req.body));
+        } catch (err){
+            console.error(err);
         }
-    }))
-} catch (err){
-    console.error(err);
-}
-});
+    });
 
-router.put('/products/:productId', async (req, res, next) => {
-    try {
-        const uProd = {}
-        for(let key in req.body){
-            uProd[key] = req.body
+router.route('/:productId')
+    .delete(async (req, res, next) => {
+        try {
+            res.send(await Product.destroy({
+                where: {
+                    id: req.params.id
+                }
+            }))
+        } catch (err) {
+            console.error(err);
         }
-        const product = await Product.update( uProd, {
-            where: {
-                id: req.params.id
+    })
+    .put(async (req, res, next) => {
```

Comment:
I feel like this route could be simplified to just use req.body instead of uProd